client/scss/utilities/_root-scope-classes.scss

```diff
@@ -292,6 +292,14 @@
   border-radius: $border-radius-unit;
 }
 
+.rounded-top-left {
+  border-top-left-radius: $border-radius-unit;
+}
+
+.rounded-bottom-right {
+  border-bottom-right-radius: $border-radius-unit;
+}
```

Comment:
We could create a class of `.round-opposites`, but I am fully aware I started this over abstraction thing...

Reply:
I dilly-dallied on this – ultimately I just knew that as soon as we created a `round-opposites` class, there would be a requirement to e.g. only round the top-right corner, so settled on the two classes.